go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/eleduardo/grpcox
```

Comment:
please use `github.com/gusaul/grpcox` instead

Reply:
@gusaul thanks for catching that!! it is fixed. Let me know what you think!